mlflow_faculty/mlflow_converters.py

```diff
@@ -77,27 +78,35 @@ def faculty_run_to_mlflow_run(faculty_run):
         "",  # source version
         lifecycle_stage,
     )
-    run_data = RunData()
+    run_data = RunData(tags=faculty_tags_to_mlflow_tags(faculty_run.tags))
     run = Run(run_info, run_data)
     return run
 
 
-def mlflow_run_metric_to_faculty_run_metric(mlflow_run_metric):
-    return FacultyMetric(
-        key=mlflow_run_metric.key,
-        value=mlflow_run_metric.value,
-        timestamp=mlflow_timestamp_to_datetime_seconds(
-            mlflow_run_metric.timestamp
-        ),
-    )
+def mlflow_metrics_to_faculty_metrics(mlflow_metrics):
```

Comment:
Why did you change these to handle lists instead of single objects? AFAICT all the other converters convert single objects, which seems like a clearer API.

Reply:
I realised that we always want to convert tags never a single tag. The idea of single tag is kind of meaningless.

Same applies to Metrics and Params

Reply:
> The idea of single tag is kind of meaningless

Not sure that's true.

In general, it seems more versatile to have the converters act on individual objects, not collections.

Reply:
We could leave it as it and then expose the single methods as and when they're needed?

Reply:
Pascal and I agreed to revert to the single-entity converters - can discuss why when you're back in the office. For now I've picked up this PR so we can push it through.